test/ssltest_old.c

```diff
@@ -1318,11 +1312,9 @@ int main(int argc, char *argv[])
             app_verify_arg.app_verify = 1;
         } else if (strcmp(*argv, "-proxy") == 0) {
             app_verify_arg.allow_proxy_certs = 1;
-        } else if (strcmp(*argv, "-test_cipherlist") == 0) {
-            test_cipherlist = 1;
         }
 #ifndef OPENSSL_NO_NEXTPROTONEG
-        else if (strcmp(*argv, "-npn_client") == 0) {
+          else if (strcmp(*argv, "-npn_client") == 0) {
```

Comment:
This file has this ifdef pattern all-over. Since this is deprecated code, rather than fixing all of it, I'd prefer to just continue to migrate the test cases over until this file is empty - is that ok?

The whitespace change here was probably just editor autocorrect.